### PR TITLE
fix: update to latest api changes

### PR DESCRIPTION
**Pull request by MatanBobi**
This PR fixes the example to work with the latest vscode insiders version using `chatParticipants` in `package.json` and `sendChatRequest` instead of `requestLanguageModelAccess` and `makeChatRequest`.
Hope I didn't miss anything but this example now works locally for me :)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -19,10 +19,22 @@
   },
   "enabledApiProposals": [
     "chatParticipant",
-    "languageModels"
+    "languageModels",
+    "defaultChatParticipant"
```
Comment (roblourens) [inline package.json]: Thanks, but it shouldn't have this proposal

File: src/extension.ts [hunk 2]
```diff
@@ -33,10 +33,10 @@ export function activate(extContext: vscode.ExtensionContext) {
 		if (git && typeof value === 'string') {
 			let fullBranchName = value.match(/Current Branch name: (.*)/i)![1];
 			const branchName = fullBranchName.split('/')[1] || fullBranchName;
-			const access = await vscode.lm.requestLanguageModelAccess(LANGUAGE_MODEL_ID);
-			const promptRequest = access.makeChatRequest([
-				new vscode.LanguageModelSystemMessage('You write creative prompts for an AI image generator. The user will give a short phrase, and you must generate a prompt for DALL-E based on that phrase. Don\'t forget to include the art style for the image. For example, it could be an oil painting, a photograph, a cartoon, a charcoal drawing, or something else. Reply with the prompt and no other text.'),
-				new vscode.LanguageModelUserMessage(branchName),
+			// const access = await vscode.lm.sendChatRequest(LANGUAGE_MODEL_ID);
```
Comment (roblourens) [inline src/extension.ts]:
```suggestion
```

Comment (roblourens) [reply src/extension.ts]: Looks like this was left over

**Comment by roblourens** [comment]
Thanks!

File: package.json (hunk 1 above)
Comment (MatanBobi) [reply package.json]: I was under that impression too, but for some reason VSCode insiders didn't allow me to run the extension without adding that proposal...

File: src/extension.ts (hunk 2 above)
Comment (MatanBobi) [reply src/extension.ts]: Oh that's my bad. Sorry!

File: package.json (hunk 1 above)
Comment (roblourens) [reply package.json]: Not sure, it worked fine for me.

Comment (MatanBobi) [reply package.json]: You're right. Tested it again without that proposal and everything works.
Sorry :)

**Comment by Iambarry1** [comment]
> This PR fixes the example to work with the latest vscode insiders version using `chatParticipants` in `package.json` and `sendChatRequest` instead of `requestLanguageModelAccess` and `makeChatRequest`.
> 
> Hope I didn't miss anything but this example now works locally for me :)

**Pull request merged**